jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -132,6 +134,17 @@ pipeline {
                                                 string(name: 'AGENT_LABEL', value: AGENT_X64)
                                             ]
 
+                                        unstash "buildInfo_manifest"
+                                        buildInfoYaml(
+                                            componentName: COMPONENT_NAME,
+                                            inputManifest: "buildInfo.yml",
```

Comment:
Input = output?

Reply:
@dblock This is actually taking `buildInfo.yml` as input since we initialize it in the previous stage.